### PR TITLE
Patch3

### PR DESCRIPTION
**Pull request by epplugins**

### CODE REVIEW

File: hello.py [hunk 1]
```diff
@@ -15,11 +15,11 @@
 # I forgot:
 df = df.copy() # Nice hack.
 
-for j in range(1000):
+for j in range(2000):
```
Comment (epf2) [inline hello.py:18]: Are you sure?

**Review by epf2 — CHANGES_REQUESTED** [review]

File: hello.py (hunk 1 above)
Comment (epplugins) [reply hello.py:18]: yes

File: hello.py [hunk 2]
```diff
@@ -15,11 +15,11 @@
...
     print(j)
 
 # We need a new funtion.
 
-def newf():
-    print("my new function")
-    return True
\ No newline at end of file
+def newf(a):
+    print("My new function.")
```
Comment (epf2) [inline hello.py:24]: Remove stop point

**Review by epf2 — CHANGES_REQUESTED** [review]

File: hello.py (hunk 2 above)
Comment (epplugins) [reply hello.py]: done

**Review by epf2 — APPROVED** [review]

**Pull request merged**